input_dir/default_config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "data_dir": "../pacta-data/2023Q4",
-  "portfolio_path": "input_dir/default_portfolio.csv",
-  "output_dir": "output_dir",
+  "data_dir": "/pacta-data/2023Q4",
+  "portfolio_path": "/workflow.pacta/input_dir/default_portfolio.csv",
+  "output_dir": "/home/workflow-pacta/output_dir",
```

Comment:
I don't understand why `/home/...` here, but I assume you've tested it and it works

Reply:
This is the standard LInus user directory. Useful primarily because it's a directory known to be owned by the user (so correct permissions in the container).